_layouts/download.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+---
+
+<div class="container">
+  {{ content }}
+
+  <ul class="tab-bar" id="tab-bar">
+    {% for platform in site.platforms %}
+    <li>{{ platform.name }}</li>
+    {% endfor %}
+  </ul>
+
+  <div id="tab-content">
+    {% for platform in site.platforms %}
+    <div class="hidden">{{ platform.content }}</div>
```

Comment:
Make sure that all content is available if js don't run.

Reply:
Added additional separator behind `noscript` and added screenshot for the nojs variant.